src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -730,6 +740,80 @@ public async Task PostImportAsync()
+
+            if (blittableJson.TryGet(nameof(DatabaseSmugglerOptions.OperateOnDatabaseRecordTypes), out string operateOnDatabaseRecordTypes) == false)
+                throw new InvalidOperationException($"{nameof(DatabaseSmugglerOptions.OperateOnDatabaseRecordTypes)} field can not be empty");
+
+            DynamicJsonValue djv = null;
+            if (Enum.TryParse(typeof(DatabaseItemType), operateOnTypes, out _) == false)
+            {
+                CheckClientVersion();
+
+                var itemsTypes = DatabaseItemType.None;
+                var types = operateOnTypes.Split(", ");
+
+                foreach (var type in types)
+                {
+                    if (Enum.TryParse(typeof(DatabaseItemType), type, true, out var result) == false || result is DatabaseItemType existing == false)
+                        continue;
+
+                    itemsTypes |= existing;
+                }
+
+                djv = new DynamicJsonValue(blittableJson)
+                {
+                    [nameof(DatabaseSmugglerOptions.OperateOnTypes)] = itemsTypes
+                };
+            }
+
+            if (Enum.TryParse(typeof(DatabaseRecordItemType), operateOnDatabaseRecordTypes, out _) == false)
+            {
+                CheckClientVersion();
+
+                var databaseRecordItemsTypes = DatabaseRecordItemType.None;
+                var types = operateOnTypes.Split(", ");
+
+                foreach (var type in types)
+                {
+                    if (Enum.TryParse(typeof(DatabaseRecordItemType), type, true, out var result) == false || result is DatabaseRecordItemType existing == false)
+                        continue;
+
+                    databaseRecordItemsTypes |= existing;
+                }
+
+                if (djv == null)
+                {
+                    djv = new DynamicJsonValue(blittableJson)
+                    {
+                        [nameof(DatabaseSmugglerOptions.OperateOnDatabaseRecordTypes)] = databaseRecordItemsTypes
+                    };
+                }
+                else
+                {
+                    djv[nameof(DatabaseSmugglerOptions.OperateOnDatabaseRecordTypes)] = databaseRecordItemsTypes;
+                }
+            }
+
+            if (djv == null)
+                return;
+
+            blittableJson.Modifications = djv;
+            using (var old = blittableJson)
+            {
+                blittableJson = context.ReadObject(blittableJson, "item-types");
+            }
+        }
+
+        private void CheckClientVersion()
+        {
+            if (RequestRouter.TryGetClientVersion(HttpContext, out var version) == false || version.Build <= RavenVersionAttribute.Instance.BuildVersion)
+                throw new InvalidOperationException("Could not find the client version or the build version is equal to/older than the current version");
```

Comment:
include the `version.Build` and `RavenVersionAttribute.Instance.BuildVersion` in the message

Reply:
done